packages/helm-charts/oracle/templates/oracle.cronjob.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata: 
+  name: {{ .Release.Name }}
+  labels: 
+    app: oracle
+    chart: oracle
+    release: {{ .Release.Service }}
+    component: oracle
+spec:
+  schedule: "{{ .Values.oracleSchedule }}"
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      backoffLimit: 1
+      template:
+        spec:
+          initContainers:
+          - name: get-current-price
+            image: gcr.io/celo-testnet/oracle:{{ .Release.Namespace }}
```

Comment:
Ah I think I get it now- should we set the repository and tag here similar to what we do for geth or celotool by having it depend on .env variables?

Reply:
Oops! Thank you! This is where those unused envVars were supposed to go.